features/custom-user-agent.json

```diff
@@ -0,0 +1,19 @@
+{
+    "_meta": {
+        "description": "Custom user agents used by our native apps that override the default user-agent string.",
+        "sampleExcludeRecords": {
```

Comment:
It's about time we drop this key IMO, it's not really relevant anymore.

Reply:
Perhaps as a follow up or LHF? Feels weird to drop that from all features in this PR.

Reply:
Yeah agreed